integration/confidential/lib.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+# Copyright (c) 2021, 2022 IBM Corporation
+# Copyright (c) 2022 Red Hat
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This provides generic functions to use in the tests.
+#
+set -e
```

Comment:
I'd consider using this set of options, as they've become pretty standard for Kata now and make the scripts a lot more robust:

```bash
set -o errexit
set -o errtrace
set -o nounset
set -o pipefail

[ -n "${DEBUG:-}" ] && set -o xtrace
```